### ISSUE TITLE
Minified React error #143

### ISSUE THREAD

**Issue by qerim** [metabase/metabase#10191]
Upgraded to the latest and encountered thiserror on homepage:
`Error: Minified React error #143; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=143 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.`

`TypeError: e is null`

Cleared cache and cookies, no luck.

Metabase version: 0.32.9
Metabase hosting environment: Elastic Beanstalk
Browser: Firefox 67.0.2 x64
Platform: Windows

**Comment by flamber** [comment]
Hi @qerim 
Are you running the official 0.32.9 or your own version?
Are you using React Developer Tools in Firefox, and having it activated for the site?

**Comment by tlrobinson** [comment]
@qerim I'm unable to reproduce this. Can you provide any other details? Are there any more logs? Does it happen when you're logged in or logged out?

**Comment by qerim** [comment]
@flamber the official, which I download from this link: [https://downloads.metabase.com/v0.32.9/metabase-aws-eb.zip](https://downloads.metabase.com/v0.32.9/metabase-aws-eb.zip) and uploaded it onto AWS beanstalk.

I don't have React developers tools installed.

Same on Chrome (75.0.3770.90 x32, no extensions installed), but with extra errors:

![image](https://user-images.githubusercontent.com/3625009/59631644-3d117700-9140-11e9-838e-4a34c8d31262.png)

@tlrobinson I logged in and out on both browsers and cleared cache + hard reload, same error. 

I am able to navigate to Admin section, to Troubleshooting section, and then when I attempt to Exit Admin, the troubleshotting body content remains:

![image](https://user-images.githubusercontent.com/3625009/59632428-1b18f400-9142-11e9-9525-b4440c510e19.png)

**Comment by flamber** [comment]
@qerim If you disable X-Rays (Admin > Settings > General > bottom) does that change anything?

A forum user reported something similar, but without confirming if there was a browser console error.
https://discourse.metabase.com/t/groups-are-not-able-to-se-any-dashboard/6493

**Comment by qerim** [comment]
@flamber thanks for the fast response. Yes I had X-ray set to off. Setting it back to on got rid of those errors :)

However, it would be nice if I can turn those back to off again as 1st time Metabase users feel overwhelmed when they first log in.

Thank you

**Comment by flamber** [comment]
It is confirmed that it's the same error in the forum. I have not been able to reproduce this.
Are you using LDAP authentication and/or Oracle as data source?
Do you see any warnings/errors in the Metabase log, when it's user login and it just spins?

**Comment by naveenh1** [comment]
We are able to reproduce the issue , whenever X-Ray setting is set to Off we get the same error. Enabling it error goes of (or we dont see loading icon at the landing page).

We use Metabase authentication which is default

**Comment by flamber** [comment]
@tlrobinson Steps to reproduce:
- Clean database and disable X-Rays and try viewing Homepage
or
- Disable X-Rays and don't have any pinned dashboards in Our Analytics (`root`)

Seems like something in Overworld is triggering this error, but I cannot see what it could be.

Error in built-hot:
```
Uncaught Invariant Violation: React.Children.only expected to receive a single React element child.
    at invariant (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:71928:15)
    at Object.onlyChild [as only] (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:183211:52)
    at LoadingAndErrorWrapper.render (http://localhost:8080/app/dist/app-main.hot.bundle.js?008cdce55f40e1a6cbb1:45155:41)
    at LoadingAndErrorWrapper.<anonymous> (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:151284:37)
    at http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:136966:21
    at measureLifeCyclePerf (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:136246:12)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:136965:25)
    at ReactCompositeComponentWrapper._renderValidatedComponent (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:136992:32)
    at ReactCompositeComponentWrapper.performInitialMount (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:136532:30)
    at ReactCompositeComponentWrapper.mountComponent (http://localhost:8080/app/dist/vendor.hot.bundle.js?008cdce55f40e1a6cbb1:136428:21)
```

**Comment by TicklesTheBrain** [comment]
I'm catching the same error in the console, with the same stuck loading gif. Everything was fine on 0.32.8, it was definitely the update to 0.32.9 that caused this. We're using LDAP for auth. Let me know if you need any more details on this.

**Comment by valtlfelipe** [comment]
Same issue here on version `0.32.9`. Happens only when I disable X-Rays.

**Comment by oddgames-david** [comment]
Happening to me also on chrome/edge on 0.32.9 and 0.33preview with xrays disabled using docker builds for both.

```
vendor.bundle.js?4b9c2174bc22535ad2bd:1 Uncaught Invariant Violation: Minified React error #143; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=143 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at t.exports (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:705072)
    at Object.t.exports [as only] (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1925106)
    at t.value (http://analytics.oddgames.com.au:3000/app/dist/app-main.bundle.js?4b9c2174bc22535ad2bd:5:128323)
    at u._renderValidatedComponentWithoutOwnerOrContext (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1997834)
    at u._renderValidatedComponent (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1997941)
    at u.performInitialMount (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1993823)
    at u.mountComponent (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1992869)
    at Object.mountComponent (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:650525)
    at u.performInitialMount (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1993984)
    at u.mountComponent (http://analytics.oddgames.com.au:3000/app/dist/vendor.bundle.js?4b9c2174bc22535ad2bd:1:1992869)
t.exports @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
t.exports @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
value @ app-main.bundle.js?4b9c2174bc22535ad2bd:5
_renderValidatedComponentWithoutOwnerOrContext @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_renderValidatedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_createInitialChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performInitialMount @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
mountComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_reconcilerUpdateChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateDOMChildren @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
receiveComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_updateRenderedComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
_performComponentUpdate @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
updateComponent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performUpdateIfNecessary @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
performUpdateIfNecessary @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
b @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
perform @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
perform @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
perform @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
w @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
closeAll @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
perform @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
batchedUpdates @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
batchedUpdates @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
dispatchEvent @ vendor.bundle.js?4b9c2174bc22535ad2bd:1
Show 122 more frames
vendor.bundle.js?4b9c2174bc22535ad2bd:1 Uncaught TypeError: Cannot read property 'getHostNode' of null
    at Object.getHostNode (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at u.getHostNode (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at Object.getHostNode (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at Object.updateChildren (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at J._reconcilerUpdateChildren (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at J._updateChildren (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at J.updateChildren (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at J._updateDOMChildren (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at J.updateComponent (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
    at J.receiveComponent (vendor.bundle.js?4b9c2174bc22535ad2bd:1)
```

**Comment by flamber** [comment]
The work-around is to pin (minimum 1) dashboard in "Our Analytics", so it shows on the frontpage, or enable x-rays.

**Comment by paulrosenzweig** [comment]
@flamber pointed out that this is resolved by #10250

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
